uaclient/status.py

```diff
@@ -120,6 +120,10 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_CONNECTIVITY_ERROR = """\
 Failed to connect to authentication server
 Check your Internet connection and try again"""
+LOG_CONNECTIVITY_ERROR_TMPL = MESSAGE_CONNECTIVITY_ERROR + ". {error}"
```

Comment:
Very much a nit, but should we use a newline instead of a `. ` here to separate the parts of the message?

Reply:
I feel like having a multiline log is hard for parsing of errors. Generally when I'm doing a quick triage/validation I expect error or warning log context  to all be on a single line (tracebacks are the only exception to this).

I was also a little uneasy about us prefacing the full MESSAGE_CONNECTIVITY_ERROR  in the error log as it already has a newline therfore making it i a bit harder to quickly grep the full context of just error messages out of the log. But in this case I thought having the same error message as seen on the console was a better breadcrumb for people trying to triage what happened in the log at the time of the error they saw in their console.